redesign-project/src/index.html

```diff
@@ -0,0 +1,54 @@
+---
+layout: layouts/default.njk
+title: Home
+---
+
+<link rel="stylesheet" href="https://use.typekit.net/hnv1vei.css">
+
+<main class="primary_content">
+    <img src="/images/EmmasMobileBG.png" alt=" " class="background">
+
+    <img src="/images/EmmasDesktopBG.png" alt=" " class="desktop_background">
+
+    <div class="rectangle_large"></div>
+
+
+    <div class="secondary_content">
+        <h2 class="h2__events">Upcoming Events</h2>
+
+        <ul class="ul__events">
+            <li><b><u>Our One-Year Anniversary Party!</u></b><br>Nov 24, 2018<br>8:00 AM – 4:00 PM</li><br>
+            <li><b><u>Kids Cook</u></b><br>Dec 1, 2018<br>9:00 AM – 10:00 AM</li><br>
+            <li><b><u>First Friday</u></b><br>Dec 7, 2018<br>6:30 PM – 8:30 PM</li><br>
+            <li><b><u>Kids' Tea Party</u></b><br>Dec 27, 2018<br>5:30 PM – 6:30 PM</li><br>
+            <li><b><u>First Friday</u></b><br>Jan 4, 2019<br>6:30 PM – 8:30 PM</li><br>
+        </ul>
+
+         <h2 class="h2__hours">Hours</h2>
+
+        <ul class="ul__hours">
+            <li><b>Monday:</b> 10:30 - 6:30</li>
+            <li><b>Tuesday:</b> Closed</li>
+            <li><b>Wednesday:</b> 10:30 - 6:30</li>
+            <li><b>Thursday:</b> 10:30 - 6:30</li>
+            <li><b>Friday:</b> 10:30 - 8:00</li>
+            <li><b>Saturday:</b> 9:00 - 4:00</li>
+            <li><b>Sunday:</b> 10:00 - 3:00</li>
+        </ul>
+        
+        <h2 class="h2__contact">Contact</h2>
+
+        <ul class="ul__contact">
+            <li><b>410 . 444 . 1718</b></li><br>
+            <li>5500 Harford Road<br>Hamilton, MD<br>21214</li>
+        </ul>
+
+        <img src="/images/PurpleInstaicon.png" alt="Instagram logo." class="insta">
```

Comment:
Both here and below the social icon images should be wrapped in `a` tags to link to those profiles.